includes/connection/class-products.php

```diff
@@ -318,7 +325,27 @@ public static function get_connection_config( $args = array() ): array {
 				'fromFieldName'  => 'products',
 				'connectionArgs' => self::get_connection_args(),
 				'resolve'        => function( $source, array $args, AppContext $context, ResolveInfo $info ) {
-					return Factory::resolve_product_connection( $source, $args, $context, $info );
+					$resolver = new PostObjectConnectionResolver( $source, $args, $context, $info, 'product' );
+
+					if ( ! empty( $args['where']['orderby'] ) ) {
```

Comment:
@kidunot89 I think we should check whether this is an array before looping through via the foreach to prevent PHP errors.

```php
if ( ! empty( $args['where']['orderby'] ) && is_array( $args['where']['orderby'] ) )
```